### DISCUSSION TITLE
🔒 Quality lock for background or pop-up streaming

### DISCUSSION THREAD

**Discussion by Alf3rez** [night/betterttv#6798] [Ideas]
Hello.

At least for me, is quite annoying that if you use some kind of picture in picture to watch a live streaming while you're doing other stuff around the browser, Twitch automatically downgrade the quality of the stream because it understands that you're not watching it, so I assume that is a feature for saving resources by their side, but for us, if we want to use it in that way is impossible.

So the idea is to introduce some option to act like the tab is active for them when it isn't, with that feature enable twitch will not downgrade the quality of the streaming to 360p anymore.

Thank you for your work.

**Comment by lunaynx** [comment]
Agreed, would be nice. They don't seem to distinguish between unfocused or PiP, so they reduce quality severely to the point where it's noticeable even in a small PiP window.

In meantime, this userscript works: https://greasyfork.org/en/scripts/383093-twitch-disable-automatic-video-downscale

**Reply by Alf3rez** [reply]
Uh, interesting, thanks for the info, I'll chek it, and yes, is just that, even in PIP small windows is pretty noticeable, hope they could integrate something similar in betterttv native

Thank you!
